book/scheduling.md

```diff
@@ -1882,15 +1900,14 @@ do that work, but that's even worse, because forcing work on the
 compositor thread will make scrolling janky unless you do even more work to
 avoid that somehow.
 
-[^servo]: The [Servo] rendering engine\index{rendering engine} uses multiple
-threads to take advantage of parallelism in style and layout, but those steps
-still block, for example, JavaScript execution on the main thread.
-
-[Servo]: https://en.wikipedia.org/wiki/Servo_(software)
+[^servo]: Some browser do use multiple threads *within* style and
```

Comment:
You don't like the Servo reference?

Reply:
Well, I'm not against, but Servo is sorta-kinda dead and meanwhile its CSS engine got adopted to Firefox (and graphics engine sorta kinda too, I think) so at first I generalized and then there didn't seem to be much need to mention Servo specifically.

Reply:
I think it's worth keeping since this was the first browser engine that used parallelism, even if it's dead. As a historical / forward-looking note at least.